constrained-voucher.mkd

```diff
@@ -148,31 +148,33 @@ Registrar/Coordinator (JRC), Manufacturer Authorized Signing Authority
 
 {::boilerplate bcp14}
 
-# Survey of Voucher Types
+# Survey of Voucher Types {#survey}
 
-{{RFC8366}} provides for vouchers that assert proximity, that authenticate the registrar  and that include different amounts of anti-replay protection.
+{{RFC8366}} provides for vouchers that assert proximity, that authenticate the Registrar and that can offer varying levels of anti-replay protection.
 
-This document does not make any extensions to the semantic meanings of vouchers, only the encoding has been changed.
+This document does not make any extensions to the semantic meanings of vouchers, only the encoding has been changed to optimize for constrained devices and networks.
 
-Time based vouchers are included in this definition, but given that constrained devices  are extremely unlikely to have accurate time, their use is very unlikely.
-Most users of these constrained vouchers will be online and will use live nonces to provide anti-replay protection.
+Time-based vouchers are supported in this definition, but given that constrained devices are extremely unlikely to have accurate time, their use is very unlikely.
+Most Pledges using these constrained vouchers will be online during enrollment and will use live nonces to provide anti-replay protection.
 
 {{RFC8366}} defined only the voucher artifact, and not the Voucher Request artifact, which was defined in {{I-D.ietf-anima-bootstrapping-keyinfra}}.
-
 This document defines both a constrained voucher and a constrained voucher-request.
 They are presented in the order "voucher-request", followed by a "voucher" response as this is
 the order that they occur in the protocol.
 
-The constrained voucher and constrained voucher request MUST be signed.
-
-The use of the two signing formats permit the use of both PKIX format certificates, and raw public keys (RPK).
+The constrained voucher request MUST be signed by the Pledge. It can sign using its IDevID X.509 certificate, or if an IDevID is not available its manufacturer-installed raw public key (RPK). 
```

Comment:
I had not imagined a scenario where the Pledge would have only a RPK.
The Registrar, yes could be RPK only, but the pledge never really needs to know/process the IDevID, but without the extensions, the Registrar can't find the MASA.  While one could have a spreadsheet of RPKs of pledges, (Eliot has a use case), I mostly think it would be better to have a spreadsheet of self-signed IDevID.